src/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py

```diff
@@ -46,6 +46,14 @@ class StableDiffusionPipeline(DiffusionPipeline):
             Model that extracts features from generated images to be used as inputs for the `safety_checker`.
     """
 
+    vae: AutoencoderKL
```

Comment:
this is a bit unrelated to the PR - let's try to put this in a different PR next time ;-)